drivers/read_fs.c

```diff
@@ -362,12 +362,13 @@ int __rt_fs_read(rt_file_t *file, unsigned int buffer, unsigned int addr, int si
   return block_size;
 }
 
-int rt_fs_seek(rt_file_t *file, unsigned int offset)
+int rt_fs_seek(rt_file_t *file, unsigned int offset, rt_event_t *event)
```

Comment:
You should not change the API, the seek operation is synchronous as it is simple.